Fastlane/deployment_lanes.rb

```diff
@@ -325,6 +325,12 @@
   UI.message "IPA saved at #{ENV['IPA_OUTPUT_PATH']}"
 end
 
+desc 'Just install the certs onto bitrise so this can build for test lab'
+lane :install_certs do
```

Comment:
```suggestion
lane :install_certificates do
```

Minor, but especially as this repository is open source, I think we'd do well to use a non-abbreviated term.